esmvaltool/cmorizers/data/utilities.py

```diff
@@ -144,6 +144,8 @@ def convert_timeunits(cube, start_year):
         real_unit = f'days since {str(start_year)}-01-01 00:00:00'
     elif cube.coord('time').units == 'days since 1950-1-1':
         real_unit = 'days since 1950-1-1 00:00:00'
+    elif cube.coord('time').units == 'days since 1970-01-01T00:00:00+00:00':
+        real_unit = 'days since 1970-01-01T00:00:00+00:00'
```

Comment:
It looks like this will not do anything (you have the exact same units twice), so you can remove it (also in the code where you use that function)